lib/configure.js

```diff
@@ -127,6 +128,15 @@ function configure (gyp, argv, callback) {
 
     // set the target_arch variable
     variables.target_arch = gyp.opts.arch || process.arch || 'ia32'
+    log.verbose('build/' + configFilename, 'target arch: ' + variables.target_arch)
+
+    // set the target_platform variable
+    variables.target_platform = process.platform
+    log.verbose('build/' + configFilename, 'target platform: ' + variables.target_platform)
+
+    // set the target modules version
+    variables.target_modules_version = nodeModulesVersion(nodeDir)
+    log.verbose('build/' + configFilename, 'modules version: ' + variables.target_modules_version)
```

Comment:
Can you wrap lines at 80 columns?  I realize there are long lines elsewhere but I'd like to avoid introducing new ones.